intranet/apps/printing/admin.py

```diff
@@ -6,7 +6,15 @@
 
 
 class PrintJobAdmin(admin.ModelAdmin):
-    list_display = ('time', 'printer', 'user', 'file', 'num_pages', 'page_range', 'printed')
+
+    def formatted_page_range(self, obj):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/tjhsst-ion/ion/file/2606998060/issues/source?bid=3212528&fileBranchId=3299315#l10)